### ISSUE TITLE
Bug: Sass build error contains an incorrect line when error is from a file referenced in injectGlobalPaths

### ISSUE THREAD

**Issue by pr3tori4n** [ionic-team/stencil-sass#20]
**Description**
If there's a compilation error in the sass, it outputs that error to the console. The error prints out the file, an error description, the line number, and the code from that line. When using the injectGlobalPaths option, if any of the SCSS files referenced by that option or any SCSS files imported by these files contains an error, the build errors and prints the error to the terminal window. The error description is accurate, and the line number seems to be reflect the line number in the injected file, **but the filename is wrong and the sample code that's contains the error is grabbed from the same incorrect file.**

```
//stencil config
plugins:[
    sass({
      injectGlobalPaths: ['src/globalStyles/variables.scss']
    })
  ] 
```
```
//src/globalStyles/variables.scss
/*Emptly line 1*/
/*Emptly line 2*/
$badSyntax
```
```
//src/components/component/component.scss
select {
    border-color: blue;
}
```

![image](https://user-images.githubusercontent.com/12614215/58906971-03615a80-86c2-11e9-8f06-db07c721a870.png)

**Actual Result:**
The file name and the code snippet in the error printed to the console are point to the component sass file and not the injected file where the error actually occurred. 
**Expected Result:**
When compiling sass encounters an error, the error prints the injected file or the imported file that contains the error, and prints the correct reference to the offending code from that file.

**Comment by pr3tori4n** [comment]
I'm also noticing what I think is a related issue: If there's an error in the component.scss file, the snippet of code and the Line number are also incorrect when using injectGlobalPaths. It seems there's something happening where the error is taking the line from the composed file that has the injected scss already added, rather than from source.

**Comment by chilipote** [comment]
I think it's a duplicate of

https://github.com/ionic-team/stencil-sass/issues/8

**Comment by jaredcbaum** [comment]
fixed as of v1.4.0
